### PR TITLE
fix delete saved view crash

### PR DESCRIPTION
**Pull request by manivoxel51**
## What changes are proposed in this pull request?

deleting saved views crashes since the property does not exist in state
https://voxel51.atlassian.net/browse/TEAMS-1029

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: fiftyone/server/mutation.py
Comment (brimoor) [inline fiftyone/server/mutation.py:325]: `state.view` is expensive because `DatasetView.__bool__` computes `len(view) > 0`. I think you should explicitly use `state.view is not None` instead.

**Review by brimoor — CHANGES_REQUESTED** [review]

File: fiftyone/server/mutation.py
Comment (manivoxel51) [reply fiftyone/server/mutation.py]: good to know 👍

Comment (brimoor) [reply fiftyone/server/mutation.py]: 👍 I defer to an App code owner for approval from here

**Review by ritch — APPROVED** [review]

**Pull request merged**
